### PR TITLE
CompanyPage Component Styling & Quintet Gallery Option

### PR DESCRIPTION
**Pull request by jocoso**
### What does this PR do?

This PR introduces the CompanyPage component into Phoenix. CompanyPage is the template used on pages on the about '/about' pages on our site. 

### Any background context you want to provide?

Screen Shot 2019-07-29 at 4 06 14 PM

Screen Shot 2019-08-22 at 2 58 07 PM

Screen Shot 2019-08-22 at 2 58 18 PM

### What are the relevant tickets/cards?

Refs [#167297536](https://www.pivotaltracker.com/n/projects/2019313/stories/167297536)

### Checklist

* [ ] Documentation added for new features/changed endpoints.
* [ ] Added appropriate feature/unit tests.
* [ ] Added screenshot to PR description of related front-end updates on **small** screens.
* [ ] Added screenshot to PR description of related front-end updates on **medium** screens.
* [ ] Added screenshot to PR description of related front-end updates on **large** screens.

### CODE REVIEW

**Comment by lkpttn** [comment]
Some screenshots of the updated design: 
Screen Shot 2020-01-08 at 14 45 32
Screen Shot 2020-01-08 at 14 45 22
Screen Shot 2020-01-08 at 14 45 46

File: resources/assets/scss/gallery-grid.scss [hunk 1]
```diff
@@ -17,6 +18,21 @@
...
+  }
+
+  @include media($large) {
+    grid-template-columns: repeat(4, 1fr);
+  }
+
+  @include media($larger) {
+    grid-template-columns: repeat(5, 1fr);
+    grid-column-gap: $base-spacing;
+    grid-row-gap: $base-spacing;
```
Comment (mendelB) [inline resources/assets/scss/gallery-grid.scss]: @lkpttn Which `theme.spacing` does `$base-spacing` translate to? I think we'll need to update this as well.

Comment (weerd) [reply resources/assets/scss/gallery-grid.scss]: yep! use the tailwind `theme()` helper:

`$base-spacing` should be replaced with `theme('spacing.6')`

Comment (weerd) [reply resources/assets/scss/gallery-grid.scss]: CSS Grid classes are coming in the next Tailwind update, so we may have a better solution in the near future :)

Comment (mendelB) [reply resources/assets/scss/gallery-grid.scss]: > 6 is the value for 1.5rem === 24px / 16px base

BTW very helpful, thank you! For some reason, I never made the explicit calculation in my brain that `1rem` === `16px` as our base size!

**Comment by weerd** [comment]
@lkpttn is this still WIP or ready for 👀 ?

**Comment by mendelB** [comment]
> @lkpttn is this still WIP or ready for 👀 ?

@weerd this should be ready for review now! Updated the title

**Comment by mendelB** [comment]
https://github.com/DoSomething/phoenix-next/pull/1559/commits/ae9a3b40ba9d0512c06af3075cca21fe5dc5630c is an update since we saw the Cypress beta referral test failing due to a default referral campaign link on the beta page being a prod campaign. So this adds a dev option for development environments

File: resources/assets/constants/index.js [hunk 2]
```diff
@@ -42,8 +42,10 @@ export const REFERRAL_CAMPAIGN_IDS = [
   '9037',
 ];
 
-// The Referral Campaign ID defaults to the Teens for Jeans campaign.
-export const DEFAULT_REFERRAL_CAMPAIGN_ID = '9037';
+// The Referral Campaign ID defaults to the Teens for Jeans campaign,
+// or a test campaign for development environments.
+export const DEFAULT_REFERRAL_CAMPAIGN_ID =
+  process.env.NODE_ENV !== 'development' ? '9037' : '9001';
```
Comment (katiecrane) [inline resources/assets/constants/index.js]: ah the ol' hardcoding, how timely!

**Pull request merged**